ops/configuration/variables/env-sample

```diff
@@ -25,8 +26,10 @@ COMPOSE_PROJECT_NAME=deployment
 # the variables will be merged into a .secrets file
 # Replace <Your fork name here> with urlencoded name of the fork
 
+REPO_NAME="<Your fork name here>"
 GROUP_VARIABLES_URL="https://gitlab.com/api/v4/groups/gigascience/variables?per_page=100"
```

Comment:
To get the group variables from gitlab organisation `fork` to avoid the `401 Unauthorized`
GROUP_VARIABLES_URL="https://gitlab.com/api/v4/groups/gigascience%2FForks/variables?per_page=100"

Reply:
Hi @kencho51,

There are three levels of hierarchical variable storages: gigascience, gigascience/forks and gigascience/forks/*-gigadb-website.

GROUP_VARIABLES_URL should keep the url as it is, since its the topmost level url for variables that are common to all contexts, while FORK_VARIABLES_URL is for the mid-level and PROJECT_VARIABLES_URL is for the lowest level (or more accurately the most specific level)


The 401 error you refer to is you not having the appropriate permissions. We could add a note in the README about the correct permission, I'd need @pli888 to tell me what he did to fix your error, so I can add in the README.

Reply:
>The 401 error you refer to is you not having the appropriate permissions. We could add a note in the README about the correct permission, I'd need @pli888 to tell me what he did to fix your error, so I can add in the README.

@rija I had to give @kencho51 `owner` role for him to access GitLab variables in the `Forks` group. `owner` seems to high a role to me. I need to read the GitLab documentation to understand how permission to access variables are managed.